test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/PartialTagHelperTest.cs

```diff
@@ -605,6 +606,57 @@ public void ResolveModel_ThrowsWhenNullModelAndForProvided()
                 () => tagHelper.ProcessAsync(tagHelperContext, output));
             Assert.Equal(expected, exception.Message);
         }
+        
+        [Fact]
+        public void Optional_DefaultValue_IsFalse()
```

Comment:
Can nuke this test.

Reply:
Thank you for the review, I deleted the test.